### ISSUE TITLE
[1.10] Causes crash due to conflict with real terrain generation mod and Vampirism

### ISSUE THREAD

**Issue by demfels** [Team-RTG/Realistic-Terrain-Generation#1225]
## Versions
- Minecraft:   1.10                 
- Forge:  12.18.3.2422          
- Vampirism:        1.2.3           
-RTG version: 1.33.664                          

Occurs during loading game (not world).

Crash report here: https://pastebin.com/7qj3cX8B

exception from Realistic Terrain Generation (RTG)
Caused by: java.lang.ExceptionInInitializerError
	at rtg.world.biome.realistic.vampirism.RealisticBiomeVAMPBase.addBiomes(RealisticBiomeVAMPBase.java:61)

**Comment by srs-bsns** [comment]
The name of the Cursed Earth block has changed from `cursedEarth` to `cursed_earth` between versions without a major version bump (this is a breaking change, tsk tsk.)...
https://github.com/TeamLapen/Vampirism/blob/v1.2.3-1.10/src/main/java/de/teamlapen/vampirism/blocks/BlockCursedEarth.java#L23

which is being statically set without any checks...
https://github.com/Team-RTG/Realistic-Terrain-Generation/blob/v1.10.2-4.2.0.0-snapshot-5/src/main/java/rtg/world/biome/realistic/vampirism/RealisticBiomeVAMPVampireForest.java#L25

**Comment by srs-bsns** [comment]
@demfels The latest version of Vampirism that you can use with `RTG-1.10.2-4.2.0.0-snapshot-5` is [Vampirism-1.10.2-1.1.2.jar](https://github.com/TeamLapen/Vampirism/releases/download/v1.1.2/Vampirism-1.10.2-1.1.2.jar)

This bug isn't likely to get fixed any time soon.

**Comment by maxanier** [comment]
Yeah, I changed that in preparation of MC1.11 to allow easier development of both branches. Sorry, this is indeed a breaking change. Did not think/remember there were any mods depending on this. Some (null) checks when working with other mods can't hurt though :P 

BTW: I have noticed the vampirism forest is not supported in RTG for MC 1.11 anymore? Guess I was too slow with updating. Should be easy to readd though, I guess. At least on Vampirism's side nothing changed regarding the biome (except the block renaming 🙈 ) **EDIT**: did not notice there isn't a MC 1.11 version out yet

@demfels 1.1.x should be quite similar to 1.2.x feature wise, mostly these internal changes
